### PR TITLE
Amazon Pharmacy demo prep, org settings package

### PR DESCRIPTION
**Pull request by jknapp25**
- extract org settings into dedicated package
- add org settings and branding
- add ability to surface mail order in patient
- add amazon pharmacy as mail order option
- zzpeds branding
- zzpeds send to patient on

### CODE REVIEW

**Comment by jknapp25** [comment]
Got a little work to make sure the app is getting settings correctly

File: apps/patient/src/components/StatusStepper/index.tsx [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
 import t from '../../utils/text.json'
-import { formatAddress } from '../../utils/general'
+import { FulfillmentType } from '../../utils/models'
 
-export const pickupStates = ['SENT', 'RECEIVED', 'READY', 'PICKED_UP']
-export const courierStates = ['SENT', 'FILLING', 'DELIVERING', 'DELIVERED']
+export const STATES = {
+  courier: ['SENT', 'FILLING', 'IN_TRANSIT', 'DELIVERED'],
+  pickup: ['SENT', 'RECEIVED', 'READY', 'PICKED_UP'],
+  mailOrder: ['SENT', 'FILLING', 'DELIVERING', 'SHIPPED']
```
Comment (samkotlove) [inline apps/patient/src/components/StatusStepper/index.tsx:9]: Should these be 
```
  mailOrder: ['SENT', 'FILLING', 'SHIPPED', 'DELIVERED']
```
?

File: packages/settings/src/lib/photon.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    federated: true
+  },
+  // ZZPeds
+  org_pxcJ7Dlclcsg0OJw: {
+    ...defaultSettings,
+    logo: 'zzpeds_logo.png',
+    accentColor: '#5271ff',
+    sendToPatient: true
+  }
+};
```
Comment (samkotlove) [inline packages/settings/src/lib/photon.ts:98]: Way cleaner 😍

**Review by samkotlove — APPROVED** [review]
One small comment but this looks great!

File: apps/patient/src/components/BrandedOptions.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+import t from '../utils/text.json'
+import { BrandedPharmacyCard } from './BrandedPharmacyCard'
+
+interface Props {
+  type: 'mailOrder' | 'courier'
+  options: string[]
+  location: string
+  selectedId: string
+  handleSelect: Function
```
Comment (captDaylight) [inline apps/patient/src/components/BrandedOptions.tsx:11]: `handleSelect: (id: string) => void`

File: apps/patient/src/components/BrandedPharmacyCard.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+import capsuleLogo from '../assets/capsule_logo.png'
+// @ts-ignore
+import amazonPharmacyLogo from '../assets/amazon_pharmacy.png'
+// @ts-ignore
+import altoLogo from '../assets/alto_logo.svg'
+
+interface Props {
+  pharmacyId: string
+  selectedId: string
+  handleSelect: Function
```
Comment (captDaylight) [inline apps/patient/src/components/BrandedPharmacyCard.tsx:21]: `handleSelect: (id: string) => void`

File: apps/patient/src/components/BrandedPharmacyCard.tsx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  Card,
+  CardBody,
+  Image,
+  SlideFade,
+  Text,
+  VStack,
+  useBreakpointValue
+} from '@chakra-ui/react'
+
+// @ts-ignore
```
Comment (captDaylight) [inline apps/patient/src/components/BrandedPharmacyCard.tsx:11]: I think we can get rid of these ts-ignore on images if we place a `assets.d.ts` file at the root of patient `/src`:

``` ts
// asset.d.ts
declare module '*.png' {
  const content: string
  export default content
}

declare module '*.svg' {
  const content: string
  export default content
}
```

https://graffino.com/til/hEvDjQa4au-how-to-import-images-in-typescript

**Review by captDaylight — APPROVED** [review]

**Comment by captDaylight** [comment]
org settings package is great idea!

File: apps/patient/src/components/BrandedOptions.tsx (hunk 3 above)
Comment (jknapp25) [reply apps/patient/src/components/BrandedOptions.tsx:11]: fixed

File: apps/patient/src/components/BrandedPharmacyCard.tsx (hunk 4 above)
Comment (jknapp25) [reply apps/patient/src/components/BrandedPharmacyCard.tsx:21]: fixed

File: apps/patient/src/components/BrandedPharmacyCard.tsx (hunk 5 above)
Comment (jknapp25) [reply apps/patient/src/components/BrandedPharmacyCard.tsx:11]: nice! 🙏

Comment (jknapp25) [reply apps/patient/src/components/BrandedPharmacyCard.tsx:11]: rad site as well

**Pull request merged**